OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -73,6 +73,9 @@ public class MobileInfo : ConditionalTraitInfo, IMoveInfo, IPositionableInfo, IF
 		[Desc("Allow multiple (infantry) units in one cell.")]
 		public readonly bool SharesCell = false;
 
+		[Desc("Does this actor block other actors?")]
+		public readonly bool OccupiesSpace = true;
```

Comment:
This really isn't very nice, IMO.  Can we please make the detector `Immobile` instead?

Reply:
Immobile doesn't implement IPositionable. IPositionable is required by Parachutable. Parachutable is required to move the detector to ground height.
And implementing IPositionable on Immobile was quite tricky last time I tried.